suites/tipset/rewards_penalties.go

```diff
@@ -178,6 +178,64 @@ func TestMinerRewardsAndPenalties(t *testing.T, factory state.Factories) {
 		td.AssertBalance(builtin.BurntFundsActorAddr, gasPenalty)
 	})
 
+	t.Run("miner penalty exceeds declared gas limit for BLS message", func(t *testing.T) {
+		td := builder.Build(t)
+		defer td.Complete()
+
+		miner := td.ExeCtx.Miner
+		tb := drivers.NewTipSetMessageBuilder(td)
+		bb := drivers.NewBlockBuilder(td, td.ExeCtx.Miner)
+
+		alice, _ := td.NewAccountActor(drivers.BLS, acctDefaultBalance)
+
+		gasPrice := int64(2)
+		gasLimit := int64(128)
```

Comment:
Put the gasPenalty number next to this one, and compute the gas limit by subtracting from the penalty. This ensures that we can see that one is less than the other, and a distracted programmer updating the hardcoded gas value in the future won't accidentally break the point of this test.